app/templates/people/person/personal-information/index.hbs

```diff
@@ -48,65 +48,92 @@
-            {{/if}}
-          </:left>
-          <:right as |Item|>
-            {{#if
-              @model.person.mandatories.firstObject.contacts.firstObject.telephone
-            }}
-              <Item>
-                <:label>Telefoonnummer</:label>
-                <:content>
-                  {{@model.person.mandatories.firstObject.contacts.firstObject.telephone}}
-                </:content>
-              </Item>
-            {{/if}}
-            <Item>
-              <:label>GSM nummer</:label>
-              <:content />
-            </Item>
-          </:right>
-        </Card.Columns>
-      </:card>
-    </DataCard> --}}
+              {{#if contact.primaryContact.email}}
+                <Item>
+                  <:label>E-mail</:label>
+                  <:content>
+                    {{contact.primaryContact.email}}
+                  </:content>
+                </Item>
+              {{/if}}
+              {{#if contact.primaryContact.contactAddress.fullAddress}}
+                <Item>
+                  <:label>Adres</:label>
+                  <:content>
+                    {{contact.primaryContact.contactAddress.fullAddress}}
+                  </:content>
+                </Item>
+              {{/if}}
+            </:left>
+            <:right as |Item|>
+              {{#if contact.role}}
+                <Item>
+                  <:label>Positie</:label>
+                  <:content>
+                    {{#if (eq contact.type "minister")}}
+                      <AuLink
+                        @route="people.person.positions.minister"
+                        @models={{array @model.person.id contact.id}}
+                      >
+                        {{contact.role}}
+                      </AuLink>
+
+                    {{else}}
+
+                      <AuLink
+                        @route="people.person.positions.mandatory"
+                        @models={{ array
+                         @model.person.id
+                          contact.id
+                        }}
+                      >
+                        {{contact.role}}
+                      </AuLink>
+                    {{/if}}
+                  </:content>
+                </Item>
+              {{/if}}
+
+              {{#if contact.primaryContact.telephone}}
+                <Item>
+                  <:label>Primair telefoonnummer</:label>
+                  <:content>
+                    {{contact.primaryContact.telephone}}
+                  </:content>
+                </Item>
+              {{/if}}
+              {{#if contact.secondaryContact.telephone}}
+                <Item>
+                  <:label>Secundair telefoonnummer</:label>
+                  <:content>
+                    {{contact.secondaryContact.telephone}}
+                  </:content>
+                </Item>
+              {{/if}}
+            </:right>
+          </Card.Columns>
+        </:card>
+      </DataCard>
+    {{/each}}
+
```

Comment:
The design shows a message here that guides users to the positions page if they want to see inactive positions. Maybe add a todo here so we are less likely to forget about it when implementing that page?

Reply:
It seems the positions page already exist, so we can link to it already.

Reply:
I added  a comment because even though the page exist, it doesn't list the minister yet, so it might confuse the user; it's part of another ticket  I think (OP-483) 
```
    {{!-- todo when posities list pages is finished, uncomment this}}
      <div class="flex items-start">
        <span class="au-u-muted au-u-italic">
          {{@model.person.givenName}}
          {{@model.person.familyName}}
          heeft geen andere posities. Om alle posities (ook niet active) te
          bekijken, ga naar&nbsp;</span>
        <AuLink @model={{@model.person}} @route="people.person.positions">
          Posities
        </AuLink>
      </div>
    {{--}}